src/stcal/jump/jump.py

```diff
@@ -18,7 +19,13 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
                  after_jump_flag_dn1=0.0,
                  after_jump_flag_n1=0,
                  after_jump_flag_dn2=0.0,
-                 after_jump_flag_n2=0):
+                 after_jump_flag_n2=0,
```

Comment:
I assume there will be a corresponding update to jwst/jump_step.py to make these parameters available to the user?